cypress.json

```diff
@@ -5,5 +5,9 @@
   "projectId": "8r2xra",
   "defaultCommandTimeout": 10000,
   "supportFile": "cypress/support/commands.js",
-  "pluginsFile": false
+  "pluginsFile": false,
+  "env": {
+    "admin_url": "/admin/",
+    "admin_login_url": "/admin/login/"
```

Comment:
Maybe someday, not worth doing now, just an idea: do we already have something for JS to read django paths for reversing? Could cypress read that from the django website it’s testing, or from a file that a management command writes to disk?